### PR TITLE
DOCS-134/NATT-3 Update stub documentation with check and alarm descriptions

### PR DESCRIPTION
**Pull request by npawelek**
Initial commit for ongoing description work around all checks and alarms.

### CODE REVIEW

**Comment by CrashenX** [comment]
Created Jira: https://rpc-openstack.atlassian.net/browse/TURTLES-1223

File: doc/maas-checks/source/stubs/host/conntrack_count/conntrack_count_status.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+==============================================
+conntrack_count_status--{{inventory_hostname}}
+==============================================
+
+Validates returned check metrics from the local ``conntrack_count.py`` plugin
+from all nodes. If the percentage of ``nf_conntrack_count`` and
+``nf_conntrack_max`` metrics reach the 90% threshold for three successive
+intervals, a critical alarm notification is generated. This indicates the
+associated node is reaching the a point of which new connections will be
```
Comment (kencjohnston) [inline doc/maas-checks/source/stubs/host/conntrack_count/conntrack_count_status.rst]: associated node is reaching the point where....

File: doc/maas-checks/source/stubs/host/filesystem_<filesystem>/filesystem_<filesystem>_check.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+=========================================================
+Disk space used on {{filesystem}}--{{inventory_hostname}}
+=========================================================
+
+Validates check metrics from the native ``agent.filesystem`` functionality for
+all nodes. If the percentage of ``used`` and ``total`` metrics reach the 90%
+threshold for three successive intervals, a critical alarm notification is
+generated. This indicates the associated node is reaching the a point at which
```
Comment (kencjohnston) [inline doc/maas-checks/source/stubs/host/filesystem_<filesystem>/filesystem_<filesystem>_check.rst]: associated node is reaching the point where ...

**Comment by mattt416** [comment]
recheck_container_bandit_tox-test
recheck_container_pep8_tox-test

**Comment by npawelek** [comment]
recheck_container_bandit_tox-test

**Comment by shannonmitchell** [comment]
recheck_all

**Comment by shannonmitchell** [comment]
recheck_xenial_master_deploy

**Comment by cloudnull** [comment]
:-1:  this is no good

**Comment by shannonmitchell** [comment]
recheck_all

**Comment by shannonmitchell** [comment]
recheck_all

**Comment by shannonmitchell** [comment]
recheck_xenial_queens_deploy

**Pull request closed without merging**